pylint/checkers/format.py

```diff
@@ -219,11 +219,11 @@ class FormatChecker(BaseTokenChecker, BaseRawFileChecker):
         (
             "indent-string",
             {
-                "default": "    ",
+                "default": '"    "',
```

Comment:
This change thé default value in the code, it's going to cause problem. There's a mechanism in place to modify the default value for the doc but not the code, see enchent dict of available language for the spell checker or default python interpreter. (On mobile, can't link to the exact place Sorry)

Reply:
I see. Yeah, I can see how this might cause a problem.

I'm looking at the arguments for enchant. Is this the one you had in mind?

```    options = (
        (
            "spelling-dict",
            {
                "default": "",
                "type": "choice",
                "metavar": "<dict name>",
                "choices": _get_enchant_dict_choices(enchant_dicts),
                "help": _get_enchant_dict_help(enchant_dicts, PYENCHANT_AVAILABLE),
            },
        ),
...
```

I see what's happening here, with functions being passed into choices/help, but the real issue is that `default` has to be four spaces, `   `, exactly in Python. This means that the value has to be different when interpreted for the conversion to `reStructuredText`.

Does adding such functionality sound like a reasonable path forward for this PR?

Reply:
Back on a PC 😄 I meant this:

https://github.com/pylint-dev/pylint/blob/eb33f8a6c0d51be08a6e3ae6a5bff0350c4048a5/doc/exts/pylint_options.py#L41-L48

Reply:
Thanks for pointing me to this @Pierre-Sassoulas !!

Been busy with work lately, but I'm hoping to find a little time to test out this change soon. 😅